dataset/data_array.cpp

```diff
@@ -104,15 +105,26 @@ DataArray &DataArray::setSlice(const Slice &s, const Variable &var) {
   return *this;
 }
 
+DataArray DataArray::view() const {
```

Comment:
Not sure i understand what this is actually doing...

Reply:
It is just what a `default` copy constructor would do. The actual copy constructor makes copies of the meta data dicts, rather than sharing them.